apps/console/src/features/groups/components/wizard/group-assign-users.tsx

```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
`Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.` needs to be changed as `Copyright (c) 2020 - 2024, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.`

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/6190